src/protocol/context/malicious.rs

```diff
@@ -451,7 +454,10 @@ impl<'a, F: Field> ContextInner<'a, F> {
         x: Replicated<F>,
         zeros_at: ZeroPositions,
     ) -> Result<MaliciousReplicated<F>, Error> {
-        let rx = x
+        let induced_share =
+            Replicated::new(x.left().get_induced_value(), x.right().get_induced_value());
+
+        let rx = induced_share
```

Comment:
In protocol 4.15 of https://eprint.iacr.org/2019/386.pdf

Step 3: Randomization of inputs: 

"For each input wire sharing [vj ] (where j ∈ {1, . . . , M}), the parties locally
compute the induced share [[vj ]] = f([vj ], 0, . . . , 0). Then, the parties call Fbmult on [[rb]] and [[vj ]] to receive
[[rb· vj ]]."